source/concepts/recording.rst

```diff
@@ -25,6 +25,81 @@ Each stream is stored in a separate file.
 By convention, recordings of the frame stream have the ``.traj`` file extension,
 while recordings of the shared state stream  have the ``.state`` file extension.
 
+Recording format
+----------------
+
+The current version of the file format is version 2.
```

Comment:
I think we can remove this line, as the current version is already stated below in the bullet points

Reply:
Done in commit [52dc633](https://github.com/IRL2/nanover-docs/pull/113/commits/52dc633153e8d43308b50314cf888181b9394a23)